features/src/main/feature/camel-features.xml

```diff
@@ -522,16 +522,15 @@
         <feature version='${project.version}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-base64/${project.version}</bundle>
     </feature>
-<!--    <feature name='camel-bean-validator' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle>mvn:org.hibernate.validator/hibernate-validator/${hibernate-validator-version}</bundle>-->
-<!--        <bundle>mvn:jakarta.validation/jakarta.validation-api/${jakarta-validation-api-version}</bundle>-->
-<!--        <bundle>mvn:org.jboss.logging/jboss-logging/${jboss-logging-version}</bundle>-->
-<!--        <bundle>mvn:com.fasterxml/classmate/${classmate-version}</bundle>-->
-<!--        <bundle>mvn:org.glassfish/jakarta.el/${jakarta-el-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel.karaf/camel-bean-validator-osgi/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-bean-validator/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-bean-validator' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jakarta-validation</feature>
+        <bundle>mvn:org.hibernate.validator/hibernate-validator/${hibernate-validator-version}</bundle>
+        <bundle>wrap:mvn:org.jboss.logging/jboss-logging/${jboss-logging-version}</bundle>
```

Comment:
We don't need the wrap protocol as it is already an OSGI bundle

Reply:
Done